lib/src/chart/base/axis_chart/axis_chart_painter.dart

```diff
@@ -26,6 +28,153 @@ abstract class AxisChartPainter<D extends AxisChartData> extends BaseChartPainte
 
     drawBackground(canvas, size);
     drawGrid(canvas, size);
+    drawAxisTitles(canvas, size);
```

Comment:
I understand the issue. Can we keep the current implementation of `drawAxisTitles()` in the `AxisChartPainter`? Otherwise I would just copy&paste this huge function into every painter subclass.

Reply:
This way if any AxisChart wants to have their special implementation of the axis titles, they can just override it, but each one has to call it during `paint()` on their own.

Reply:
Ok, let them be there.
Thanks!